src/main/resources/tags/help/luckperms.tag

```diff
@@ -1,6 +1,9 @@
 type: text
 aliases: lp
 issues: is not a valid username/uuid
+title: :tools: LuckPerms Not Recognizing Bedrock Players
+color: warning
+image: https://ibb.co/0jfyb65J
```

Comment:
This image won't embed properly (I don't think). It should just be a Discord link, which will work fine. Or, replace this with https://i.ibb.co/QF6vwT0m/250718-23h52m16s-screenshot.png